documentation/sphinx/source/api-c.rst

```diff
@@ -720,7 +726,7 @@ Applications must provide error handling and an appropriate retry loop around th
 
 .. function:: FDBFuture* fdb_transaction_get_approximate_size(FDBTransaction* tr)
 
-   Retrieves the approximate transaction size so far in the returned future, which is the summation of the estimated size of mutations, read conflict ranges, and write conflict ranges. This can be called multiple times before transaction is committed.
+   Retrieves the approximate transaction size so far in the returned future, which is the summation of the estimated size of mutations, read conflict ranges, and write conflict ranges. The size can be obtained by calling :func:`fdb_future_get_int64()` with the returned `FDBFuture` object. This can be called multiple times before transaction is committed.
```

Comment:
We use a specific pattern in other functions for describing these functions that return a future (e.g. see `fdb_transaction_get_versionstamp`). I think we should follow the same pattern here.